include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -473,8 +470,8 @@ __parallel_transform_scan(_ExecutionPolicy&& __exec, _Range1&& __rng1, _Range2&&
             __kernel_1,
 #endif
             sycl::nd_range<1>(__n_groups * __wgroup_size, __wgroup_size), [=](sycl::nd_item<1> __item) {
-                __local_scan(__item, __n, __local_acc, __rng1, __rng2, __wg_sums_acc, __size_per_wg, __wgroup_size,
-                             __iters_per_witem, __init);
+                __local_scan(__item, __n, __local_acc, __rng1, __rng2, __wg_sums_acc, __size_per_work_group,
```

Comment:
Could you please align the names as well? I mean `__wg_sums_acc` -> `__work_group_sums_acc`, `__wgroup_size` -> `__work_group_size` and so on.